spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -793,7 +770,6 @@ The following built-in annotations define a `Literal` static nested class to sup
 * `jakarta.enterprise.inject.Any`
 * `jakarta.enterprise.inject.Default`
 * `jakarta.enterprise.inject.Specializes`
-// TODO move to Full?
```

Comment:
Since the API JAR will be the same and these will be part of it anyway, I don't it is worth moving them into a separate section.